thirdeye-ui/src/app/routers/welcome/welcome.router.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React, { FunctionComponent, lazy, Suspense } from "react";
+import { Navigate, Outlet, Route, Routes } from "react-router-dom";
+import { PageNotFoundPage } from "../../pages/page-not-found-page/page-not-found-page.component";
+import { AppLoadingIndicatorV1 } from "../../platform/components";
+import { AppRouteRelative } from "../../utils/routes/routes.util";
+
+const WelcomeLandingPage = lazy(() =>
+    import(
+        /* webpackChunkName: "welcome-landing-page" */ "../../pages/welcome-landing-page/welcome-landing-page.component"
```

Comment:
don't forget to also change the comment name

Reply:
Is this comment not right?

Reply:
oh yeah you're correct, its good now